### PR TITLE
fix wrong jetty-all config in pom.xml

### PR DESCRIPTION
**Pull request by flaming-archer**
### :pencil: Description
Exclude jetty-all from hive-service in pom.xml of core module, because it makes spring start fail and makes `WaggleDanceIntegrationTest` fail.

### :link: Related Issues

### CODE REVIEW

**Comment by flaming-archer** [comment]
The following error will occur：
`java.lang.NoClassDefFoundError: org/eclipse/jetty/io/Connection$Listener
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)`

**Comment by flaming-archer** [comment]
Because the class `Connection` has not inner class `Listener`
`public class JettyConnectionMetrics extends AbstractLifeCycle implements Connection.Listener {

    private final MeterRegistry registry;`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [3.11.3] - 2023-08-02
+### Changed
+- Exclude jetty-all from hive-service in pom.xml of core module, because it makes spring start fail and makes `WaggleDanceIntegrationTest` fail.
```
Comment (patduin) [inline CHANGELOG.md:3]: I can't really reproduce this and the tests run fine in our GH actions, can you tell what environment you are running that's causing this issue?

**Review by patduin — APPROVED** [review]
Thanks for this! It made us realize the github actions are not running tests correctly at the moment. Something we'll look into.

**Comment by patduin** [comment]
> @patduin hello, `Java CI / Package and run all tests (pull_request) Failing after 1m`， this mistake doesn't seem to have anything to do with my pr, right? If possible, pls merge it into the master branch. This will make me happy and be my first PR for this, hahaha.
> 
> In the future, our company will implement our logic based on this.

No it looks like there is an issue with the Github actions regarding jar signing I'll look into that. 
Your PR looks good, thanks for contributing!

**Comment by flaming-archer** [comment]
> > @patduin hello, `Java CI / Package and run all tests (pull_request) Failing after 1m`， this mistake doesn't seem to have anything to do with my pr, right? If possible, pls merge it into the master branch. This will make me happy and be my first PR for this, hahaha.
> > In the future, our company will implement our logic based on this.
> 
> No it looks like there is an issue with the Github actions regarding jar signing I'll look into that. Your PR looks good, thanks for contributing!

Looking forward to being merged.

File: waggle-dance-core/pom.xml [hunk 2]
```diff
@@ -178,6 +178,10 @@
           <groupId>org.eclipse.jetty</groupId>
           <artifactId>jetty-runner</artifactId>
         </exclusion>
+        <exclusion>
```
Comment (patduin) [inline waggle-dance-core/pom.xml:181]: This dependency is also pulled in by hive-common shouldn't it be excluded from here as well?

Comment (flaming-archer) [reply waggle-dance-core/pom.xml:185]: Yes, I think you are right ,should exclude from hive-common as well.

**Review by abhimanyugupta07 — APPROVED** [review]

**Pull request merged**